### DISCUSSION TITLE
[마감 긴박] 모바일 Lighting 오류

### DISCUSSION THREAD

**Discussion by diddntjd99** [naverz/zepeto-studio-kor#103] [5. Q&A]
안녕하세요,
현재 아이폰에서 Lighting 설정의 Environment Reflections 설정에서 해상도가 맞지 않는건지, 이유를 모르겠지만 반사체 색상이 누락되는 것을 확인할 수 있습니다. 또한, SkyBox를 이용하여 하늘 색을 변경해보면 위와같이 모든 기종의 모바일에서 반사체가 누락됩니다. 당장 오늘 오후 6시에 마감이기 때문에 빠른 시간 내로 해결 받을 수 있으면 좋겠습니다.

**Comment by cheolhui-lee** [comment]
안녕하세요.
아래 이미지와 같이 기본적인 리플렉션은 되는 것으로 확인을 했는데,
혹시 어떻게 안되는 것인지 자세한 내용 설명 부탁드립니다.
![screenshot_20211119-165300](https://user-images.githubusercontent.com/3862666/146662108-73f1fe21-7b36-4d59-8d24-95b091de5377.png)

**Reply by diddntjd99** [reply]
@cheolhui-lee
![2](https://user-images.githubusercontent.com/71220570/146664554-73687f07-2c3e-482a-b161-d318e4ac4be6.png)
검은색으로 보이는 부분이 물인데 아예 시커멓게 보이고, 월드의 밝기도 전체적으로 어둡게 보입니다

![3](https://user-images.githubusercontent.com/71220570/146664583-274ddc3b-2dfc-456f-8a3e-cca80635c000.jpg)
이게 정상적인 갤럭시 기종 핸드폰에서의 모습입니다.

**Reply by cheolhui-lee** [reply]
일부 기기에서만 비정상 상황이라는 것일까요?
일단 어둡게 보이거나 하는 부분은 씬의 라이트 설정을 조절해보시는 것이 좋을 것 같습니다.

**Reply by diddntjd99** [reply]
@cheolhui-lee 
네 둘 다 똑같은 라이트를 사용한 상황입니다. 근데, 갤럭시에선 실행했을 때 밑의 사진처럼 보이고, 위는 아이폰으로 실행했을 때 모습인데
테스트를 해보니, 리플랙션 부분을 custom으로 설정해서 본인들이 만든 스카이박스로 배치하니까 갤럭시, 아이폰 모두 정상적으로 보였지만 전체적인 건물이 비춰쳐서 이와 같은 방법을 사용하지 못하였습니다.
리플랙션을 skybox로 설정해놓고 해상도가 다르면 저렇게 뜨는것도 확인이 되었는데, 갤럭시와 아이폰의 해상도가 다른지 궁금합니다

**Reply by diddntjd99** [reply]
![image](https://user-images.githubusercontent.com/71220570/146665043-0dfe08c1-ba16-4b10-8242-b92a294377c9.png)
이게 위에서 말한 skybox로 설정한 것이고,

![image](https://user-images.githubusercontent.com/71220570/146665057-d26cc26e-2afa-4c09-9597-a0abd60ee5e6.png)
이게 custom으로 저희가 만든 스카이박스를 넣은 설정 모습입니다

**Reply by cheolhui-lee** [reply]
환경반사 해상도를 제페토 앱에서 따로 제어하는 부분은 없습니다.
유니티에서 리플렉션 연산하는 부분이 플랫폼 별로 다를 수는 있을 것 같아요.

일단 당장 기본 스카이박스를 사용하시려면 둘 다 정상적으로 노출될 수 있는 해상도를 찾아서 설정하는 방법으로 대응을 할 수 밖에는 없어보이네요.

해당 내용 관련하여 자세한 내용은 내일 출근하면 확인이 가능할 것 같습니다.

**Reply by diddntjd99** [reply]
@cheolhui-lee 오늘이 제출이라 해결이 안된다면 추후에 업데이트하는 식으로 해야겠네요..

**Reply by diddntjd99** [reply]
답변 감사합니다
